docs/reference/doofinder.cartitemstatsparams.index.md

```diff
@@ -0,0 +1,13 @@
+<!-- Do not edit this file. It is automatically generated by API Documenter. -->
+
+[Home](./index.md) &gt; [doofinder](./doofinder.md) &gt; [CartItemStatsParams](./doofinder.cartitemstatsparams.md) &gt; [index](./doofinder.cartitemstatsparams.index.md)
+
+## CartItemStatsParams.index property
+
+Name of the indice the item belongs to. i.e. "product" \*

```

Comment:
According to @carlosescri (and of course, the English grammar, the singular form for "Indices" is "index"). See https://doofinder.slack.com/archives/G010SDC0BSN/p1690362264404459

Reply:
done